src/message/update-auth-password/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { plainToInstance, Type } from 'class-transformer';
+import { ChargingStation } from '../../pages/charging-stations/ChargingStation';
+import { IsNotEmpty } from 'class-validator';
+import { GenericForm } from '../../components/form';
+import { Form } from 'antd';
+import { Constructable } from '../../util/Constructable';
+import { BaseRestClient } from '../../util/BaseRestClient';
+import { showError, showSucces } from '../util';
+import { MessageConfirmation } from '../MessageConfirmation';
+
+export interface UpdateAuthPasswordProps {
+  station: ChargingStation;
+}
+
+enum UpdateAuthPasswordRequestProps {
+  password = 'password',
+  setOnCharger = 'setOnCharger',
+  stationId = 'stationId',
+}
+
+export class UpdateAuthPasswordRequest {
+  @IsNotEmpty()
+  password!: string | null;
+
+  @IsNotEmpty()
+  setOnCharger!: boolean | null;
+}
+
+// TODO: Remove this function and use the updated triggerMessageAndHandleResponse function.
+export const triggerMessageAndHandleResponseWithDataURL = async <T,>(
+  url: string,
+  responseClass: Constructable<T>,
+  data: any,
+  responseSuccessCheck: (response: T) => boolean,
+) => {
+  try {
+    const client = new BaseRestClient();
```

Comment:
@eddygrebt please align this with the changes in https://github.com/citrineos/citrineos-operator-ui/pull/31 and use the triggerMessageAndResponse function from that pr with the 'isData' boolean"

Reply:
looks like this was updated 👍